deploy/helm/sumologic/values.yaml

```diff
@@ -2491,20 +2491,20 @@ opentelemetry-operator:
       repository: public.ecr.aws/sumologic/opentelemetry-operator
     collectorImage:
       repository: "public.ecr.aws/sumologic/sumologic-otel-collector"
-      tag: "0.108.0-sumo-0"
+      tag: "0.114.0-sumo-0"
     autoInstrumentationImage:
       java:
         repository: public.ecr.aws/sumologic/autoinstrumentation-java
-        tag: 1.26.0
+        tag: 2.10.0
```

Comment:
this version has been tested, it works eventhough it is a major version change.

@fguimond @amdprophet this should address a customer ask to support Otel SDK 0.44.1

Reply:
nvm, I may have spoke too soon about the version upgrade.

Reply:
I'll do this upgrade in a separate PR and run the tests.